packages/runtime/native/shared/Css_types.ml

```diff
@@ -190,7 +189,7 @@ module Length = struct
     | `add of calc_value * calc_value
     | `sub of calc_value * calc_value
     | `mult of calc_value * calc_value
-    | `div of calc_value * float
+    | `div of calc_value * calc_value
```

Comment:
heads up, this doesn't seem right. You can't divide by something that isn't a number.

Reply:
Ok, it's a bit more complex than this:

```
Current implementations require that for the * and / operators, one of the operands has to be unitless. For /, the right operand must be unitless. For example font-size: calc(1.25rem / 1.25) is valid but font-size: calc(1.25rem / 125%) is invalid.
```

https://developer.mozilla.org/en-US/docs/Web/CSS/calc

Reply:
Yeah the formal grammar is a bit misleading.
```
<calc-product> = <calc-value> [ [ '*' |  '/' ] <calc-value>]*
```
 maybe we can have <unitless-calc-value>

Reply:
maybe later, i think float should be sufficient for now